src/server/scripts/Northrend/Naxxramas/boss_noth.cpp

```diff
@@ -157,6 +157,7 @@ class boss_noth : public CreatureScript
 
         void JustDied(Unit*  killer) override
         {
+			me->NearTeleportTo(nothPosition.GetPositionX(), nothPosition.GetPositionY(), nothPosition.GetPositionZ(), nothPosition.GetOrientation(), true);
```

Comment:
I would like to check if Noth is in the Balcony before teleporting him.

Reply:
@Nefertumm feel free to remove `To be merged` if you want to prevent this PR to be merged until your review comment is addressed

Reply:
@ArashGorge